### PR TITLE
Add 4080248P7

### PR DESCRIPTION
**Pull request by vojtechvit**
Tested with my 4080248P7 light (Philips LCF003) using this script:

```javascript
const tz = require('zigbee-herdsman-converters/converters/toZigbee');
const extend = require('zigbee-herdsman-converters/lib/extend');
const ota = require('zigbee-herdsman-converters/lib/ota');

const hueExtend = {
    light_onoff_brightness_colortemp_color: (options={}) => ({
        ...extend.light_onoff_brightness_colortemp_color({supportsHS: true, ...options}),
        toZigbee: extend.light_onoff_brightness_colortemp_color({supportsHS: true, ...options})
            .toZigbee.concat([tz.hue_power_on_behavior, tz.hue_power_on_error]),
    }),
};

module.exports = {
    zigbeeModel: [ 'LCF003', '4080248P7' ],
    model: '4080248P7',
    vendor: 'Philips',
    description: 'Hue Signe floor light',
    meta: { turnsOffAtBrightness1: true },
    extend: hueExtend.light_onoff_brightness_colortemp_color(),
    ota: ota.zigbeeOTA,
};
```

Relates to https://github.com/Koenkk/zigbee-herdsman-converters/issues/3380

### CODE REVIEW

File: devices/philips.js [hunk 1]
```diff
@@ -1525,6 +1525,15 @@ module.exports = [
         extend: hueExtend.light_onoff_brightness_colortemp_color(),
         ota: ota.zigbeeOTA,
     },
+    {
+        zigbeeModel: [ 'LCF003', '4080248P7' ],
+        model: '4080248P7',
+        vendor: 'Philips',
+        description: 'Hue Signe floor light',
+        meta: { turnsOffAtBrightness1: true },
+        extend: hueExtend.light_onoff_brightness_colortemp_color(),
```
Comment (Koenkk) [inline devices/philips.js:1534]: Please add the color temperature range as described on https://www.zigbee2mqtt.io/advanced/support-new-devices/01_support_new_devices.html#_3-adding-converter-s-for-your-device

Comment (vojtechvit) [reply devices/philips.js]: added

**Comment by vojtechvit** [comment]
corresponding zigbee1mqtt docs PR: https://github.com/Koenkk/zigbee2mqtt.io/pull/1141

**Comment by Koenkk** [comment]
thanks!

**Pull request merged**